locales/es_ES/pcsx2_Main.po

```diff
@@ -2069,67 +1949,49 @@ msgid "Recompiler"
 msgstr "Recompilador"
 
 #: pcsx2/gui/Panels/CpuPanel.cpp:121
-msgid ""
-"Performs just-in-time binary translation of 64-bit MIPS-IV machine code to "
-"x86."
-msgstr ""
-"Realiza una traducción binaria al vuelo del código de máquina de 64 bits "
-"MIPS-IV a x86."
+msgid "Performs just-in-time binary translation of 64-bit MIPS-IV machine code to x86."
+msgstr "Realiza una traducción binaria al vuelo del código de máquina de 64 bits MIPS-IV a x86."
 
 #: pcsx2/gui/Panels/CpuPanel.cpp:127
 msgid "Pretty slow; provided for diagnostic purposes only."
 msgstr "Muy lento; solo útil para realizar diagnósticos."
 
 #: pcsx2/gui/Panels/CpuPanel.cpp:130
-msgid ""
-"Performs just-in-time binary translation of 32-bit MIPS-I machine code to "
-"x86."
-msgstr ""
-"Realiza una traducción binaria al vuelo del código de máquina de 32 bits "
-"MIPS-I a x86."
+msgid "Performs just-in-time binary translation of 32-bit MIPS-I machine code to x86."
+msgstr "Realiza una traducción binaria al vuelo del código de máquina de 32 bits MIPS-I a x86."
 
 #: pcsx2/gui/Panels/CpuPanel.cpp:154
 msgid "Enable EE Cache (Slower)"
 msgstr "Activar caché EE (Lento)"
 
 #: pcsx2/gui/Panels/CpuPanel.cpp:154
 msgid "Interpreter only; provided for diagnostic"
-msgstr "Solo para intérprete, usado para realizar diagnósticos."
+msgstr "Solo para intérprete, usado para realizar diagnósticos"
 
 #: pcsx2/gui/Panels/CpuPanel.cpp:170 pcsx2/gui/Panels/CpuPanel.cpp:226
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:134 pcsx2/gui/Panels/VideoPanel.cpp:294
 msgid "Restore Defaults"
 msgstr "Reiniciar ajustes predeterminados"
 
 #: pcsx2/gui/Panels/CpuPanel.cpp:186
-msgid ""
-"Vector Unit Interpreter. Slow and not very compatible. Only use for "
-"diagnostics."
-msgstr ""
-"Intérprete de la Unidad Vectorial. Lento y no muy compatible. Solo para "
-"diagnósticos."
+msgid "Vector Unit Interpreter. Slow and not very compatible. Only use for diagnostics."
+msgstr "Intérprete de la Unidad Vectorial. Lento y no muy compatible. Solo para diagnósticos."
 
 #: pcsx2/gui/Panels/CpuPanel.cpp:188
 msgid "microVU Recompiler"
-msgstr "Recompilador microVU"
+msgstr "recompilador microVU"
 
 #: pcsx2/gui/Panels/CpuPanel.cpp:189
-msgid ""
-"New Vector Unit recompiler with much improved compatibility. Recommended."
-msgstr ""
-"Nuevo recompilador de la Unidad Vectorial con compatibilidad muy mejorada. "
-"Recomendado."
+msgid "New Vector Unit recompiler with much improved compatibility. Recommended."
+msgstr "Nuevo recompilador de la Unidad Vectorial con compatibilidad muy mejorada. Recomendado."
 
 #: pcsx2/gui/Panels/CpuPanel.cpp:192
 msgid "superVU Recompiler [legacy]"
-msgstr "Recompilador superVU (legado)"
+msgstr "recompilador superVU [legado]"
```

Comment:
Here as well.

Reply:
I changed these to better reflect the behaviour of the English version, but I can revert them.

Reply:
`Recompilador superVU [legado]` would be good :)

Reply:
Done. I changed some more strings from capitalized to uncapitalized for the same reason, you can have a look at it if you wish and I can also revert them if you feel its necessary :)